mkosi/__init__.py

```diff
@@ -1821,6 +1826,7 @@ def build_image(args: MkosiArgs, config: MkosiConfig) -> None:
         install_grub_bios(state, partitions)
         make_image(state, split=True)
 
+        normalize_mtime(state.root,  state.config.environment.get("SOURCE_DATE_EPOCH"))
```

Comment:
This one shouldn't be needed if we call normalize_mtime() before we call make_image()